packages/ListBoxWithTags/src/components/Pill/Pill.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Close from "@paprika/icon/lib/Times";
+import * as sc from "./Pill.styles";
+
+const propTypes = {
+  onRemove: PropTypes.func.isRequired,
+  children: PropTypes.node.isRequired,
+  size: PropTypes.oneOf(["small", "medium", "large"]),
```

Comment:
Constants. On lines 13 and 39, too.

Reply:
yeah I'm fine with this duplication, reads correctly wherever you are and doesn't provide any extra effort that would be adding it to a constant. You know me I'm against micro abstractions.

Reply:
Also the Pill will be their own component so this will get refactor :)